From 014dc43447fa553e875984ac4a528e5916cc9e06 Mon Sep 17 00:00:00 2001
From: Carlos Santos <unixmania@gmail.com>
Date: Sat, 29 Feb 2020 22:41:30 -0300
Subject: [PATCH] runlevels: do not add agetty.tty[1-6] if MKSYSVINIT=yes

Buildroot starts a single getty, according to the system configuration.
Also tty[1-6] may not exist (e.g. embedded devices with serial consoles
only).

Signed-off-by: Carlos Santos <unixmania@gmail.com>
[Adam: update for 0.50]
Signed-off-by: Adam Duskett <adam.duskett@amarulasolutions.com>
---
 tools/meson_runlevels.sh | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/tools/meson_runlevels.sh b/tools/meson_runlevels.sh
index 1473d4e..eee256b 100755
--- a/tools/meson_runlevels.sh
+++ b/tools/meson_runlevels.sh
@@ -91,11 +91,5 @@ if ! test -d "${DESTDIR}${shutdowndir}"; then
 		ln -snf "${init_d_dir}/$x" "${DESTDIR}${shutdowndir}/$x"
 	done
 fi
-if test "${sysvinit}" = yes && test "${os}" = Linux; then \
-	for x in tty1 tty2 tty3 tty4 tty5 tty6; do
-		ln -snf "${init_d_dir}/agetty" "${DESTDIR}/${init_d_dir}/agetty.$x"
-		ln -snf "${init_d_dir}/agetty.$x" "${DESTDIR}/${defaultdir}/agetty.$x"
-	done;
-fi
 
 ln -snf "${rc_libexecdir}"/sh/functions.sh "${DESTDIR}/${init_d_dir}"
-- 
2.18.2

